### PR TITLE
Feature [WIP] : bank account management (create bank account)

### PR DESCRIPTION
**Pull request by HellBus1**
1. Create database connection using go/database & setup postgre driver
2. Add new route /v1/bank/account to create bankAccount & bankAccountController.go to interact with service
3. Create bankAccountService.go to interact with database instance
4. Add unit test for bankAccountController & bankAccountService
5. Add payload validator for BankAccountRequest

### CODE REVIEW

**Comment by codeOraaange** [comment]
Monggo alignment for keep bankAccountRouter or adjust to userRouternya ya @HellBus1. cc @martjellino

**Comment by HellBus1** [comment]
> Monggo alignment for keep bankAccountRouter or adjust to userRouternya ya @HellBus1. cc @martjellino

maksudnya gimana ya han? sori agak lost wkkw

**Comment by HellBus1** [comment]
i'll merge it if it acceptable

File: router/router.go [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-	router.GET("/health-check", func(c *gin.Context) {
-		c.JSON(200, gin.H{
-			"message": "success",
-		})
+	router.Use(func(c *gin.Context) {
+		c.Set("db", db)
+		c.Next()
 	})
 
+	userRouter := router.Group("/v1/bank/account")
```
Comment (martjellino) [inline router/router.go:19]: just change a little bit to be bankAccountRouter instead of userRouter :)

**Review by martjellino — APPROVED** [review]
not problem

**Pull request merged**